sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_attribution_v1/query.sql

```diff
@@ -1,16 +1,193 @@
 -- Query for telemetry_derived.active_users_aggregates_attribution_v1
+CREATE TEMP FUNCTION normalize_adjust_network(key STRING) AS (
+  (
+    CASE
+    WHEN
+      key NOT IN (
+        '',
```

Comment:
Are we sure we want to keep an empty space for the field here?

Reply:
The reason to keep it here is so that it doesn't classify as `Other` which would not be correct.

Reply:
@kik, I talked to the stakeholder, they want to keep the field empty for analysis.

Reply:
👍

Reply:
Then let's leave it for now, but in general I think it would be good for us as analytics team to have a discussion how we think these kind of values (empty string) should be handled across the org.